yarn.lock

```diff
@@ -11028,16 +11017,6 @@ mkdirp@1.x, mkdirp@^1.0.4:
   dependencies:
     minimist "0.0.8"
 
-mobx-react-lite@^1.5.2:
-  version "1.5.2"
-  resolved "https://registry.yarnpkg.com/mobx-react-lite/-/mobx-react-lite-1.5.2.tgz#c4395b0568b9cb16f07669d8869cc4efa1b8656d"
-  integrity sha512-PyZmARqqWtpuQaAoHF5pKX7h6TKNLwq6vtovm4zZvG6sEbMRHHSqioGXSeQbpRmG8Kw8uln3q/W1yMO5IfL5Sg==
-
-mobx@^5.13.0:
```

Comment:
Always glad to see this guy gone.
I've always hated the fact `tree-list` used `mobx`, as it was always tad redundant, but since Studio and all other components used it, it never really had any impact on the bundle size.
Now, when we use some other crap that we'll probably abandon in 2022 (:trollface:), it makes sense to drop it.
FWIW, I have a PR in tree-list repo that swaps `mobx` with a tiny custom thingy -> https://github.com/stoplightio/tree-list/blob/dfc024db74b1b7de2f81d50a4d674a3729861344/src/observable.ts.
We've never merged it, but perhaps I get to merge it, so that `RequestMaker` (if it still uses tree-list) does not have that dependency included.

Reply:
FYI it's 2021, so `RequestMaker`, in the form you know it, will be gone soon, it's deprecated. 😎

 In the upcoming Elements v7 we have something new that doesn't rely on `tree-list` anymore. So if we eliminate `mobx` here, that should be enough to finally get rid of `mobx` in Elements completely. Doesn't help platform but it helps future opensource users.

Reply:
> Doesn't help platform but it helps future opensource users.

:100: